R/creation.R

```diff
@@ -243,7 +243,7 @@ require_parent_group <- function(
 #'     Primary compressor.
```

Comment:
I did a ton of clean up in here. Nothing should have changed but the duplication is basically all gone. A couple check issues are taken care of here too.